easybuild/easyconfigs/f/FreeTDS/FreeTDS-1.2-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,45 @@
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org>
+#
+# Fred Hutchinson Cancer Research Center - Seattle Washington - US
+#
+easyblock = 'ConfigureMake'
+
+name = 'FreeTDS'
+version = '1.2'
+
+homepage = "https://www.freetds.org/"
+description = """FreeTDS is a set of libraries for Unix and Linux that allows your programs to
+ natively talk to Microsoft SQL Server and Sybase databases."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['ftp://ftp.freetds.org/pub/freetds/stable']
```

Comment:
Please use the `https://` download URL instead: https://www.freetds.org/files/stable/

Reply:
updating url

Reply:
all the commands have usage messages, but no argument to access them. -h --help -v all produce error, but with a nice usage message.